src/main/java/uk/gov/hmcts/reform/idam/web/AppController.java

```diff
@@ -420,10 +420,11 @@ public ModelAndView login(@ModelAttribute("authorizeCommand") @Validated Authori
                         staleUserResetPasswordParams.remove(PASSWORD);
                         staleUserResetPasswordParams.remove(SELF_REGISTRATION_ENABLED);
                         return new ModelAndView("redirect:/reset/inactive-user", staleUserResetPasswordParams);
-                    case ACCOUNT_LINKED_TO_EXTERNAL_PROVIDER:
-                        model.addAttribute(IS_ACCOUNT_SSO_ACCOUNT, true);
-                        bindingResult.reject("Account Linked to SSO Provider");
-                        return new ModelAndView(LOGIN_VIEW, model.asMap());
+                        // redirect to the standard "invalid username/password" - see: SIDM-4810
```

Comment:
Why commented out, isn't it better to just remove it?

Reply:
👍